### PR TITLE
DEPENDABOT: Add dependabot to hocs-case-creator

### PR DESCRIPTION
**Pull request by nickmcmahon01**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
-apply plugin: 'org.springframework.boot'
-apply plugin: 'io.spring.dependency-management'
 
 group 'uk.gov.digital.ho.hocs'
 version '1.0-SNAPSHOT'
 sourceCompatibility = '11'
+
 repositories {
     mavenCentral()
-    mavenLocal()
```
Comment (LiamMacP) [inline build.gradle:26]: Should these be removed?

Comment (nickmcmahon01) [reply build.gradle:26]: I think so, there's no mavenLocal on other projects and nothing will be on drone- we should be consistent.
As for artifactory I really don't like the idea of needing a username and password to build a project... 
I don't understand why the schema is its own project at all - where is the potential for reuse(or why we have a schema!)

Comment (nickmcmahon01) [reply build.gradle:26]: happy to defer, of course

Comment (LiamMacP) [reply build.gradle:26]: The mavenLocal is so you can build the schema and push it locally so locally you don’t need the credentials. 

For it being on artifactory, it’s there because it is consumed by the HOF team. It is also what the public form is validated against and should be private to prevent misuse.

Comment (nickmcmahon01) [reply build.gradle:26]: I'll put these back, but the repo is public https://github.com/UKHomeOffice/hocs-ukvi-complaint-schema so there is little point publishing to artifactory...

Comment (LiamMacP) [reply build.gradle:26]: Good point, we can review this as part of the case-creator partial re-write.

File: lombok.config [hunk 2]
```diff
@@ -1,2 +1 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
+lombok.anyConstructor.addConstructorProperties=true
```
Comment (LiamMacP) [inline lombok.config:1]: This is a change that seems out of scope? Do we need it, what is it's benefit?

Comment (nickmcmahon01) [reply lombok.config:1]: its consistent with the other projects
https://stackoverflow.com/questions/26703645/dont-understand-constructorproperties

we'll need it when we start to use DTOs properly, but we'll forget so a good time to put it in is when we move to the same lombok as the rest of the project.

**Review by LiamMacP — APPROVED** [review]
